examples/curl-easy-report.rs

```diff
@@ -18,6 +18,11 @@ use threescalers::{
 use curl::easy::Easy;
 
 fn main() -> Result<(), threescalers::errors::Error> {
+    use std::time::{
```

Comment:
Why is this `use` here instead of at the top of the file?
In some cases, I guess this can be useful to convey that the library is only used in a specific part of the code, but this is the first line of the main :thinking:

Reply:
It's only used in `main` and I wanted the compiler to ensure there were no other `SystemTime`s floating around.